docs/contributing-to-analyses/pr-review-and-merge/index.md

```diff
@@ -1,8 +1,32 @@
-# Code review and merging
+# Code review and merging your pull request
 
-> _More content coming soon!_
+Once you have submitted your [pull request (PR)](../creating-pull-requests/index.md), your PR will be reviewed by a Data Lab team member.
+We will review your code contribution by:
 
-This section provides information about the code review process for [pull requests (PRs)](../creating-pull-requests/index.md) as well as the actions you should take after your PR is merged.
+- Checking for correctness, clarity, and reproducibility in the code
+- Evaluating methods and rationale for any proposed analysis
+- Ensuring that all necessary documentation is present and clear
 
-All PRs will undergo code review by a Data Lab member.
-PRs must be approved before they can be merged into `AlexsLemonade/OpenScPCA-analysis` repository.
+## Overview of the review process
+
+Below is a general overview of what you can expect during the code review process:
+
+1. In most cases, the first time your PR is reviewed by a Data Lab member, they will provide comments and suggestions.
+Feel free to browse some [example reviewer comments](STUB_LINK to example review comments).
+
+1. You will then need to respond to those comments.
+Follow [these guidelines when addressing review comments](STUB_LINK to addressing review comments).
+
+1. Once you have fully addressed comments from the reviewer, then you will be able to re-request a review.
+
+The above steps will repeat until the reviewer feels that all comments have been addressed adequately.
+At that time, the PR will be approved.
+Bear in mind that it is _normal and expected_ for a pull request to undergo multiple rounds of review before it is approved.
+
+## Approved pull requests
+
+Once approved, the PR will be merged into the main branch of `AlexsLemonade/OpenScPCA-analysis` by a Data Lab team member.
+
```

Comment:
Are there other S3 syncing things they need to do after PRs are merged? I don't exactly recall...

Reply:
I don't think so... or at least I don't think we've figured that out fully? So for right now I think this is sufficient?